esp/clients/ws_dfsclient/ws_dfsclient.cpp

```diff
@@ -623,8 +635,24 @@ IDFSFile *lookupDFSFile(const char *logicalName, unsigned timeoutSecs, unsigned
             throw makeStringExceptionV(-1, "eclwatch '%s': service port not defined", eclWatchName.str());
         const char *protocol = eclWatch.getPropBool("@tls") ? "https" : "http";
         serviceUrl.appendf("%s://%s:%u", protocol, result.first.c_str(), result.second);
-    }
+#else
+        static std::vector<std::string> dfsServiceUrls;
```

Comment:
statics within functions have weird semantics you almost certainly don't want (thread-safely initialised on first call).    Better for them to be outside.

Reply:
I put them here within the #else of the #ifdef _CONTAINERIZED to keep the code together, and because the non-containerized case is not the main target of this PR at the moment, this code is here primarily so I can test with the daliadmin remotetest in baremetal.
It is very unlikely anyone would hit this code but me, via daliadmin remotetest, after manually configuring a WsDfs service in the environment.